yarn.lock

```diff
@@ -5738,7 +5774,7 @@ direction@^1.0.1:
   resolved "https://registry.yarnpkg.com/direction/-/direction-1.0.3.tgz#5030e1e091e923904067d015dbaafd08f4d27d26"
   integrity sha512-8bHRqMt4w/kND19KBksE4NOJo+gIOPuiZfxQvbd6xikfKbuNBYBdLIw0hA/4lWzBaDpwpW+Olmg1BjD9+0LU2w==
 
-dnd-core@^7.0.2, dnd-core@^7.4.4:
+dnd-core@^7.4.4:
```

Comment:
Why is an older dnd-core still in yarn.lock?

Reply:
It seems it's from `react-dnd-touch-backend`:

https://github.com/redonkulus/react-dnd-touch-backend/blob/master/package.json#L38